chef_master/source/cookbook_repo.rst

```diff
@@ -116,7 +116,7 @@ This configuration file has the following settings:
         :default => 'kitty kitty'
 
 ``chef_version``
-   A range of chef-client versions that are supported by this cookbook.
+   A range of chef-client versions that are supported by this cookbook. All :ref:`version constraint operators <cookbook_version_constraints>` are applicable to this field.
```

Comment:
there might be a difference here, though which is that chef_version supports multiple version constraints, while cookbook deps are still single-constraints and we've never fixed that bug?  so `chef_version ">= 12.5.1", "< 14.0"` is legal while `depends ">= 12.5.1", "< 14.0"` is not.